node/src/main/scala/coop/rchain/node/api/package.scala

```diff
@@ -42,6 +49,12 @@ package object api {
           DeployServiceV1GrpcMonix
             .bindService(deployGrpcService, grpcExecutor)
         )
+        .keepAliveTime(keepAliveTime, TimeUnit.MILLISECONDS)
+        .keepAliveTimeout(keepAliveTimeout, TimeUnit.MILLISECONDS)
+        .permitKeepAliveTime(permitKeepAliveTime, TimeUnit.MILLISECONDS)
+        .maxConnectionIdle(maxConnectionIdle, TimeUnit.MILLISECONDS)
+        .maxConnectionAge(maxConnectionAge, TimeUnit.MILLISECONDS)
+        .maxConnectionAgeGrace(maxConnectionAgeGrace, TimeUnit.MILLISECONDS)
```

Comment:
Default values are specified in nanoseconds and here are milliseconds. A good reason to use duration. :)
https://github.com/grpc/grpc-java/blob/1c50bcafd/netty/src/main/java/io/grpc/netty/NettyServerBuilder.java#L101-L105

Reply:
It makes sense.